zipline/pipeline/loaders/blaze/core.py

```diff
@@ -952,6 +954,17 @@ def collect_expr(e):
                 columns=added_query_fields + list(map(getname, columns)),
             )
         )
+        if not materialized_deltas.empty and have_sids:
+            missing_sids = materialized_deltas[
+                ~materialized_deltas.sid.isin(assets)
+            ]
+            log.debug(
+                "Didn't find the following sids in asset index: {}".format(
```

Comment:
dont call format in the log.debug call, just pass the formatters as seperate arguments. This prevents the formatting from happening if debug logs are being dropped.

Also, I am still not sure we need this logging. it is 100% valid for there to be assets here that are not in the baseline and that is an expected case. If we really want this logging to happen downstream we might want to look at adding this in the downstream consumer of zipline.

Also noting that I am pretty sure we do not show debug logs by default so this might need to be an info wherever we agree this should go.